eng/Version.Details.xml

```diff
@@ -24,5 +24,94 @@
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
       <Sha>e5d12ecc90cad2b588e454ebec36556658981b28</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="6.0.0-preview.5.21301.17" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>5a007deadbb2636caff4656333cafc818b6b2ef6</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0-preview.5.21301.5" CoherentParentDependency="Microsoft.AspNetCore.App.Runtime.win-x64">
+      <Uri>https://github.com/dotnet/runtime</Uri>
```

Comment:
To anyone else looking: The URI's and SHA's here are arbitrary and don't matter. Maestro will update them when it next runs.

Reply:
fixed